.github/workflows/bors.yml

```diff
@@ -266,9 +266,13 @@ jobs:
       - name: check for noisy stdout lines
         id: noisy
         run: |
-          grep --after-context=1 "stdout" stdout.log && ret=0
-          grep --after-context=1 "stderr" stdout.log && new=0
-          if [ "${ret}" == "0" ] || [ "${new}"  == "0" ]; then exit 1; fi
+          buildMsgs="$(lake build Mathlib Archive Counterexamples |
+            awk '!($1 ~ "^[[?0-9]*/[?0-9]*]"){ print $0 }')"
```

Comment:
Could you add a comment explaining what this `awk` invocation is doing?

Perhaps also a comment above buildMsgs that explains we are taking advantage of lake's replay.

Reply:
Comments added: let me know if this is too much/too little!

Reply:
LGTM